### PR TITLE
All statements and queries are closed in finally statements

### PR DESCRIPTION
**Pull request by dryganets**
SQLiteCipher crashes/hangs in cases when you don't close prepared statements correctly.

In this commit, I'm moving all cursors close statements inside of finally block.
Also missing close statements are added.

### CODE REVIEW

File: src/android/src/main/java/org/pgsqlite/SQLitePlugin.java [hunk 1]
```diff
@@ -435,12 +437,7 @@ private SQLiteDatabase openDatabase(String dbname, String assetFilePath, int ope
                 cbc.error("can't open database " + ex);
             throw ex;
         } finally {
-            if (in != null) {
-                try {
-                    in.close();
-                } catch (IOException ignored) {
-                }
-            }
+           closeQuietly(in);
```
Comment (andpor) [inline src/android/src/main/java/org/pgsqlite/SQLitePlugin.java]: like this!

**Comment by andpor** [comment]
Excellent - all these make common sense. This cleanup was overdue. Great work Sergei! Do you have any more PR coming?

**Comment by dryganets** [comment]
This is all I have for now.
I pushed sqlite cipher to my forks branch. It's a very  small diff. Will do more cleanup later on.  Thank you for quick merge of changes!

**Comment by andpor** [comment]
@dryganets  Sergey - if it is a small diff maybe it would make sense to incorporate it somehow to this module...I would have no objections actually..can you suggest something?

**Comment by dryganets** [comment]
Well,  the problem is not in the Dependency size,  but in the library size. If you don't need encryption you don't want to have an extra mb to your apk.

We need to make a plugin model for Android with default implementation using system library.

**Comment by brodybits** [comment]
> @dryganets Sergey - if it is a small diff maybe it would make sense to incorporate it somehow to this module...I would have no objections actually..can you suggest something?

Also see https://github.com/andpor/react-native-sqlite-storage/issues/40#issuecomment-282312755:
> I would recommend keeping encryption separate. When publishing apps with encryption people would have to follow the guide at https://discuss.zetetic.net/t/export-requirements-for-applications-using-sqlcipher/47 and it may be even worse in places like France.

**Comment by itinance** [comment]
According to @brodybits arguments: can we leave this optional per #ifdef direction on objC-Side and something similar on android?
For those who need encryption they can activate it then easily and others don't have to ship one more MB with the app and aren't bothered by the export-requirements for apps using cipher

**Comment by andpor** [comment]
I have to think about this. @brodybits touches on import/export compliance which is quite important...

**Pull request merged**